### PR TITLE
Update php cheatsheet

### PR DESCRIPTION
**Pull request by ritwikdalmia**

### CODE REVIEW

File: CheatSheets/php-cheatsheet.md [hunk 1]
```diff
@@ -1,29 +1,123 @@
----
-title: PHP CheatSheet
-description: The most commonly used php concepts are given here.
-created: 2022-10-23
----
-
```
Comment (crescentpartha) [inline CheatSheets/php-cheatsheet.md:6]: Don't remove these lines of code. It's required.

File: CheatSheets/php-cheatsheet.md [hunk 2]
```diff
@@ -1,29 +1,123 @@
...
 ## Table of Contents
 
 - [PHP CheatSheet for Developers](#php-cheatsheet-for-developers)
   - [Escape Sequence](#escape-sequence)
+  - [Operators](#operators)
+  -  - [Operators](#operators)
```
Comment (crescentpartha) [inline CheatSheets/php-cheatsheet.md:6]:
```suggestion
  - [Operators](#operators)
```

File: CheatSheets/php-cheatsheet.md [hunk 3]
```diff
@@ -1,29 +1,123 @@
...
+|`>`|	Greater than|	$x > $y|Comparison Operators|	
+|`<`|	Less than|	$x < $y|Comparison Operators|	
+|`>=`|	Greater than or equal to|	$x >= $y|Comparison Operators|	
+|`<=`|	Less than or equal to|	$x <= $y|Comparison Operators|	
+|`<=>`|	Spaceship|	$x <=> $y	|Comparison Operators|
+|`and`|	And|	$x and $y|Logical	Operators|
+|`or`	|Or|	$x or $y	|Logical	Operators|
+|`xor`|	Xor|	$x xor $y	|Logical	Operators|
+|`&&`|	And|	$x && $y|	Logical	Operators|
+| ![image](https://user-images.githubusercontent.com/47249568/197706502-62d506bd-5596-4c5f-a1e3-7e58633c4139.png)	|Or|	$x ![image](https://user-images.githubusercontent.com/47249568/197706528-2bbe3369-5c51-4e1b-b295-c607f05bfa77.png) $y |Logical	Operators|
```
Comment (crescentpartha) [inline CheatSheets/php-cheatsheet.md:53]:
```suggestion
|  `\|`	| Or | $x \| $y | Logical Operators |	
```
| Operator | Name | Syntax | Type |
| -------- | -------- | ----------- | -------- |
|  `\|`	| Or | $x \| $y | Logical Operators |

File: CheatSheets/php-cheatsheet.md [hunk 4]
```diff
@@ -1,29 +1,123 @@
...
+|`?:	`|Ternary|	$x = expr1 ? expr2 : expr3|	Conditional Assignment Operators|
+|`??`|	Null coalescing|	$x = expr1 ?? expr2|	Conditional Assignment Operators|
+
+
+|operator|name |description|type|
+|---------|------|-----|----|
+|`++$x`|	Pre-increment|	Increments $x by one |Increment|
+|`$x++`|	Post-increment|	Returns $x, then increments $x by one	|Increment|
+|`--$x`|	Pre-decrement	|Decrements $x by one, then returns $x|decrement|	
+|`$x--`	|Post-decrement	|Returns $x, then decrements $x by one	|decrement|
```
Comment (crescentpartha) [inline CheatSheets/php-cheatsheet.md:72]: Could you divide all the above operators like `Increment & Decrement` Operators? 
For example, `Arithmetic`, `Comparison`, `Logical`, `String`, `Array`, `Conditional` Operations etc.

**Review by crescentpartha — CHANGES_REQUESTED** [review]
Need some modifications. Please, follow the comments below on what to do next. Thank you, @ritwikdalmia

**Comment by ritwikdalmia** [comment]
updated and solved all the conflicts and queries

File: CheatSheets/php-cheatsheet.md [hunk 5]
```diff
@@ -7,23 +7,424 @@ created: 2022-10-23
...
+|`.=`|	Concatenation assignment|	$txt1 .= $txt2|	
+
+## Array Operators
+
+|`+`|	Union|	$x + $y|	
+|`==`|	Equality|	$x == $y	|
+|`===`|	Identity|	$x === $y |
+|`!=`|	Inequality|	$x != $y	|
+|`<>`|	Inequality|	$x <> $y	|
+|`!==`|	Non-identity|	$x !== $y	|
```
Comment (crescentpartha) [inline CheatSheets/php-cheatsheet.md:96]: The table is broken again.

**Review by crescentpartha — CHANGES_REQUESTED** [review]

**Review by crescentpartha — APPROVED** [review]
Approved.

**Pull request merged**